core/io/http_command.hxx

```diff
@@ -176,6 +182,21 @@ struct http_command : public std::enable_shared_from_this<http_command<Request>>
       span_ = nullptr;
     }
     if (auto handler = std::move(handler_); handler) {
+      auto telemetry_recorder = app_telemetry_meter_->value_recorder(session_->node_uuid(), {});
```

Comment:
I think doing the same as done w/ the KV commands would be good for the UUID:

```
const auto& node_uuid = session_ ? session_->node_uuid() : "";
```

Reply:
I will update it to do it